### PR TITLE
[APB-704][RC] add Arn and Utr from the frontend so can be shared.

### PR DESCRIPTION
**Pull request by ralphcollett**

### CODE REVIEW

**Comment by ralphcollett** [comment]
There were some views on naming in the context of this project. Please let me know what you guys think. As is, I have just copied Arn and Utr from agent-mapping-frontend.

**Comment by frankoid** [comment]
Since the library has "mtd" in its name I think there's no need to rename Arn to MtdArn.

Also since the library's domain is MTD we can probably get away with leaving Utr as Utr for the time being because at the moment the only UTRs we are dealing with are SA and CT ones. However this may change in future as MTD expands to other regimes so there is an argument that calling it SaOrCtUtr would be more future-proof.

File: project/HmrcBuild.scala [hunk 1]
```diff
@@ -1,24 +1,19 @@
...
-  import uk.gov.hmrc.{SbtBuildInfo, ShellPrompt}
-  import uk.gov.hmrc.PublishingSettings._
 
   val appName = "agent-mtd-identifiers"
 
   lazy val microservice = Project(appName, file("."))
     .enablePlugins(SbtAutoBuildPlugin, SbtGitVersioning)
     .settings(
-      scalaVersion := "2.11.8",
+      scalaVersion := "2.11.7",
```
Comment (frankoid) [inline project/HmrcBuild.scala]: what is the reason for changing the scala version?

Comment (ralphcollett) [inline project/HmrcBuild.scala]: Accidental copy-and-paste from http-verbs as it was the first library project I could think of and there wasn't a template for creating a library. Thanks for spotting, I'll change it back.

**Pull request merged**